### DISCUSSION TITLE
莹妹卡哇伊

### DISCUSSION THREAD

**Discussion by IHYAHED** [share121/inter-knot#2425] [其他]
![116368150_p0](https://github.com/user-attachments/assets/901ddf5a-fd87-4efb-8bb8-0d878216553b)

**Comment by YuTinMIn** [comment]
可爱捏

**Comment by Genshin-Neuvillette** [comment]
豪康

**Comment by userxwn10086** [comment]
好看

**Comment by nightrain2333** [comment]
高清图爱了爱了<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >

**Comment by Lin559** [comment]
怎么保存图片啊，只能用电脑截屏再剪切吗，想用来当壁纸。
